common/src/commonMain/kotlin/com.harmony.kotlin/data/datasource/network/GenericNetworkDataSource.kt

```diff
@@ -4,21 +4,17 @@ import com.harmony.kotlin.common.thread.network
 import com.harmony.kotlin.data.datasource.DeleteDataSource
 import com.harmony.kotlin.data.datasource.GetDataSource
 import com.harmony.kotlin.data.datasource.PutDataSource
-import com.harmony.kotlin.data.error.ktor.KtorNetworkException
-import com.harmony.kotlin.data.mapper.Mapper
-import com.harmony.kotlin.data.query.*
+import com.harmony.kotlin.data.query.Query
 import io.ktor.client.HttpClient
-import io.ktor.client.features.ClientRequestException
 import io.ktor.client.request.*
 import io.ktor.http.ContentType
 import io.ktor.http.contentType
 import kotlinx.serialization.KSerializer
 import kotlinx.serialization.builtins.list
 import kotlinx.serialization.json.Json
-import kotlinx.serialization.list
 
 open class GetNetworkDataSource<T>(
-    private val url: String,
+    private val _url: String,
```

Comment:
I don't know why did you use the underscore character but I think **url** is not a good name for the property, **host** maybe is a better name.

Reply:
Because HttpRequestBuilder got a property called **url** but now I found a workaround so I leave it as it was.

url